cal/__init__.py

```diff
@@ -66,13 +66,7 @@ def home():
     return render_template('index.html', events=events)
 
 
-# TO REMOVE: Manual update route
-@app.route('/update')
-def update():
-    update_fb_events()
-    return jsonify({"success": True})
-
-
+# TO REMOVE
 @app.route('/events')
```

Comment:
this should have a decorator that depends on the `DEBUG` flag that either enables it or throws a 404